javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,15 @@
+/* exported includesSeven */
+
+function includesSeven(array) {
+  var index = [];
+  var i = 0;
+  while (i < array.length) {
+    index.push(i);
+    i++;
+  }
+  if (i === 7) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
You can simplify this a bit. Try returning `true` inside your loop if `i === 7`. After the loop (assuming the loop never returned), `return false;`